configs/mot/qdtrack/qdtrack_faster-rcnn_r101_fpn_12e_tao.py

```diff
@@ -0,0 +1,53 @@
+# model settings
+_base_ = [
+    './qdtrack_faster-rcnn_r101_fpn_24e_lvis.py',
+    '../../_base_/datasets/tao.py', '../../_base_/default_runtime.py'
+]
+model = dict(freeze_detector=True)
+img_norm_cfg = dict(
+    mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)
+train_pipeline = [
+    dict(type='LoadMultiImagesFromFile'),
+    dict(type='SeqLoadAnnotations', with_bbox=True, with_track=True),
+    dict(
+        type='SeqResize',
+        img_scale=[(1333, 640), (1333, 672), (1333, 704), (1333, 736),
+                   (1333, 768), (1333, 800)],
+        share_params=True,
+        multiscale_mode='value',
+        keep_ratio=True),
+    dict(type='SeqRandomFlip', share_params=True, flip_ratio=0.5),
+    dict(type='SeqNormalize', **img_norm_cfg),
+    dict(type='SeqPad', size_divisor=32),
+    dict(type='MatchInstances', skip_nomatch=True),
+    dict(
+        type='VideoCollect',
+        keys=['img', 'gt_bboxes', 'gt_labels', 'gt_match_indices'],
+    ),
+    dict(type='SeqDefaultFormatBundle', ref_prefix='ref'),
+]
```

Comment:
Can these be reused?

Reply:
The train data source and dataset type is different, so we have to rewrite here.

Reply:
But the `train_pipeline` can be used?